tests/test_service.py

```diff
@@ -279,6 +279,54 @@ def test_tag_build_if_multiple_rules_are_matched(
                 ]
             })
 
+    @patch('message_tagging_service.tagging_service.retrieve_modulemd_content')
+    @patch('koji.ClientSession')
+    @patch('koji.read_config')
+    def test_tag_build_with_multiple_tags(
+            self, read_config, ClientSession, retrieve_modulemd_content):
+        read_config.return_value = koji_config_krb_auth
+
+        # Because rule file specifies that destination tag uses the value of
+        # requires.platform, and there are two of those values in moduelmd,
+        # the module build should be tagged with two tags.
+        retrieve_modulemd_content.return_value = dedent('''\
+            ---
+            document: modulemd
+            version: 2
+            data:
+              name: javapackages-tools
+              stream: 1
+              version: 1
+              context: c1
+              dependencies:
+              - buildrequires:
+                  platform: [f29, f28]
```

Comment:
@mprahl Each module build is a stream expanded build. Should there be only one value in `platform`?

Reply:
Yes, you're right.